lib/instance.js

```diff
@@ -1024,7 +1025,7 @@ module.exports = (function() {
         delete defaults[this.Model._timestampAttributes.deletedAt];
       }
 
-      if (Object.keys(defaults).length) {
+      if (Object.keys(defaults).length !== 0) {
```

Comment:
Then there's really no reason to change it. It's better to keep style changes out of functional PR's so they can be discucssed seperately. I prefer the other style :)